backend/clubs/migrations/0059_recurringevent.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.1.2 on 2020-12-04 21:59
```

Comment:
You can merge these two migrations by deleting them and running `./manage.py makemigrations` again.